core/src/toga/app.py

```diff
@@ -349,6 +348,27 @@ def __init__(
     def _create_impl(self):
         return self.factory.App(interface=self)
 
+    @property
+    def paths(self):
+        """
+        Paths for known safe system locations. Using paths makes sure you have a consistent and known location to store
+        files for your app. Also, some platforms do not allow arbitrary file access to any location on disk. On Android,
+        there are many limitations when it comes to accessing file locations and the apis you can use to read from those
+        locations.
+
+        PROPERTIES:
+            **App**: Location of where the apps code is run.
+
+            **Data**: System appropriate location to store user data.
+
+            **Cache**: System cache location. Files stored in this location may be able to be accessed quicker than normal locations.
```

Comment:
This is a bit misleading. Cached files are used to optimise runtime performance, but that's not because the files load faster - cache file locations are just file locations. They're no faster to access. 

The distinction is more about whether the files should be considered a permanent store of data, or ephemeral. You should assume that any cached file can and will be purged by the operating system at any time. Your browser uses cached files so that it doesn't need to repeatedly download an asset file; but if the cached file is deleted, it can be restored by downloading the file again.